TesseractOCR/G8Tesseract.mm

```diff
@@ -819,14 +816,19 @@ - (Pix *)pixForImage:(UIImage *)image
     const UInt8 *pixels = CFDataGetBytePtr(imageData);
 
     size_t bitsPerPixel = CGImageGetBitsPerPixel(cgImage);
+    int bytesPerPixel = (int)bitsPerPixel / 8;
```

Comment:
This can't be negative and since it's more of a size anyway, I think it should be a `size_t` as well.